.ci/jobs.t/0-pr-precommit.yml

```diff
@@ -0,0 +1,18 @@
+- job:
+    name: elastic+elasticsearch+pr+precommit
+    display-name: "elastic / elasticsearch # PR precommit (experimental)"
```

Comment:
This replaces the other tests when the `>test-mute` label is present. 
@mark-vieira since your taking this Pr over, it might be a good idea to add comments explaining what the odd pr builder syntax really means